src/Illuminate/Foundation/Application.php

```diff
@@ -1019,39 +1012,6 @@ public function provideFacades($namespace)
     }
 
     /**
-     * Define a callback to be used to configure Monolog.
-     *
-     * @param  callable  $callback
-     * @return $this
-     */
-    public function configureMonologUsing(callable $callback)
```

Comment:
I suggest keeping the `configureMonologUsing`, it makes possible to use totally customized log Handlers, Formatters and Channels independently from the framework.

I use it in my case to configure how I gonna handle the app logs, sending to external services via Handlers based on log level, passing through [FingersCrossedHandler](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/FingersCrossedHandler.php) and [WhatFailureGroupHandler](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Handler/WhatFailureGroupHandler.php) and so on.

It's a so specific strategy that I doubt that will be shipped my default on Laravel. Removing this functionality is a :-1: for me.

Reply:
Well you can easily just create your own channel that attaches the handlers to your default logging? That'd be the new way to do it, a new `SuperSpecialChannel` which extends `Channel` and pushes whichever handlers you want onto the monolog instance. The entire reason for this PR is in fact because of how limited `configureMonologUsing` is

Reply:
This has to be kept, eventually you can mark it as deprecated but we can't just remove a public method.